packages/x-client/src/types/api.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable max-len */
+import { imx } from '@imtbl/generated-clients';
+
+/**
+ * Need to specifically export the classes and interfaces from the generated
+ * clients imx object for rollup to bundle them correctly.
+ */
```

Comment:
This file doesn't look ideal, but was required for rollup to bundle the classes and interfaces correctly during build. The alternative was to reexport the `imx` object, but that would have exposed a lot of unused generated code.